zerver/migrations/0641_empty_subject_for_slack_dms.py

```diff
@@ -0,0 +1,70 @@
+# Generated by Django 5.0.9 on 2024-11-08 13:02
+
+import time
+
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.db.models import Q
+
+BATCH_SIZE = 1000
```

Comment:
You want 10000; most messages will not match this filter.

Reply:
Great point which I didn't consider. I have increased the batch size in my current PR.